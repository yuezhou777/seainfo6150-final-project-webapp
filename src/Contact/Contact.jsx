import React from 'react';
import NavBar from "../NavBar/NavBar";
import styles from "./Contact.module.css";


const Contact = () => {
    return(
        <div>
            <div>
                <NavBar />
            </div>

            <div className={styles.container}>
                    <h2 className={styles.contactTitle}>Contact now to get more info:</h2>
                    <div className={styles.contactContent}>
                        <address>
                            <a href="mailto:zhou.yue2@northeastern.com" className={styles.link}>PURR PET</a><br/>
                            Written by Yue Zhou<br/>
                            Call us: +1-(206)-636-9951<br/>
                            Visit us at:<br/>
                            2442 8th Ave N<br/>
                            Washingtong<br/>
                            USA
                        </address>
                    </div>
                </div>
        </div>
    );

}

export default Contact;