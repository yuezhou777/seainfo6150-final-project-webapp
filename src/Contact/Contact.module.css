.container {
    width: 50%;
    margin: auto;
    align-self: center;
    display: flex;
    flex-flow: column;
    justify-content: center;
    align-items: center;
    font-family: 'Comic Sans MS', sans-serif;

    background: #fff;
    border: 2px orange;
    border-radius: 10px;
    box-shadow: 0px 0px 10px 10px orange;
    padding: 15px;
}

.contactTitle {
    font-size: 2em;
}

.contactContent {
    font-size: 1.2em;
    display: flex;
    justify-content: flex-start;
}

.link:hover {
    background-color: greenyellow;
}