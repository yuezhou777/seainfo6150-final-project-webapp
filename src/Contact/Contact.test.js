import React from "react";
import { render } from "@testing-library/react";
import Contact from "./Contact.jsx";
import { BrowserRouter } from 'react-router-dom';

it("renders Contact Correctly", () => {

    const { container } = render(
        <BrowserRouter>
            <Contact />
        </BrowserRouter>
    );

    expect(container).toMatchSnapshot();
});