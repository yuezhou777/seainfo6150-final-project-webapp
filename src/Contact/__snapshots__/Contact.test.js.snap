// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Contact Correctly 1`] = `
<div>
  <div>
    <div>
      <div
        class="navContainer"
      >
        <header
          class="navContainer"
        >
          <nav>
            <a
              class="navIcon"
              href="/"
            >
              PurrPet
            </a>
          </nav>
          <nav>
            <ul
              class="navList"
            >
              <li
                class="navItem"
              >
                <a
                  class="link"
                  href="/aboutus"
                >
                  About Us
                </a>
              </li>
              <li
                class="navItem"
              >
                <a
                  class="link"
                  href="/allpets"
                >
                  All Pets
                </a>
              </li>
              <li
                class="navItem"
              >
                <a
                  class="link"
                  href="/category"
                >
                  Categories
                </a>
              </li>
              <li
                class="navItem"
              >
                <a
                  class="link"
                  href="/contactus"
                >
                  Contact Us
                </a>
              </li>
            </ul>
          </nav>
        </header>
      </div>
    </div>
    <div
      class="container"
    >
      <h2
        class="contactTitle"
      >
        Contact now to get more info:
      </h2>
      <div
        class="contactContent"
      >
        <address>
          <a
            class="link"
            href="mailto:zhou.yue2@northeastern.com"
          >
            PURR PET
          </a>
          <br />
          Written by Yue Zhou
          <br />
          Call us: +1-(206)-636-9951
          <br />
          Visit us at:
          <br />
          2442 8th Ave N
          <br />
          Washingtong
          <br />
          USA
        </address>
      </div>
    </div>
  </div>
</div>
`;
